taskcat/_cli_modules/test.py

```diff
@@ -97,7 +106,8 @@ def run(  # noqa: C901
         lint_disable: bool = False,
         enable_sig_v2: bool = False,
         keep_failed: bool = False,
-        output_directory: str = './taskcat_outputs',
+        output_directory: str = "./taskcat_outputs",
+        profile: str = "default",
```

Comment:
shouldn't we add `*args`/`**kwargs` for consistency

Reply:
Good catch.